tools/nnicmd/config_utils.py

```diff
@@ -71,3 +65,44 @@ def read_file(self):
             except ValueError:
                 return {}
         return {}
+
+class Experiments:
+    '''Maintain experiment list'''
+    def __init__(self):
+        os.makedirs(HOME_DIR, exist_ok=True)
+        self.experiment_file = os.path.join(HOME_DIR, '.experiment')
```

Comment:
how about move .experiment to HOME_DIR/nni?

Reply:
I see. So suggest to rename HOME_DIR to NNICTL_HOME_DIR to dispense with any confusion.